NEWS.md

```diff
@@ -1,5 +1,7 @@
 # dbplyr (development version)
 
+* Adds support for the `.by_group` argument in `arrange()` (#115)
```

Comment:
I think this is technically a breaking change (although it makes dbplyr more consistent with dplyr) so can you please add a little more detail?